Mesh_3/include/CGAL/Mesh_3/Mesh_complex_3_in_triangulation_3_base.h

```diff
@@ -1013,7 +1013,14 @@ bbox() const
     result = result + tr_.point(vit).bbox();
   }
 
-  return result;
+  const double offsetmin = 1. - offset;
+  const double offsetmax = 1. + offset;
+  return result = Bbox_3(offsetmin * result.xmin(),
+                         offsetmin * result.ymin(),
+                         offsetmin * result.zmin(),
+                         offsetmax * result.xmax(),
+                         offsetmax * result.ymax(),
+                         offsetmax * result.zmax());
```

Comment:
This formula is wrong. It is scaling centered at `ORIGIN` whereas it should be centered at the center of the bbox.

```suggestion
  const auto offset_x = offset * (result.xmax() - result.xmin()) / 2.;
  const auto offset_y = offset * (result.ymax() - result.ymin()) / 2.;
  const auto offset_z = offset * (result.zmax() - result.zmin()) / 2.;
  return result = Bbox_3(result.xmin() - offset_x,
                         result.ymin() - offset_y,
                         result.zmin() - offset_z,
                         result.xmax() + offset_x,
                         result.ymax() + offset_y,
                         result.zmax() + offset_z);
```